engine-tests/src/main/java/org/terasology/testUtil/WorldProviderCoreStub.java

```diff
@@ -142,18 +144,24 @@ public byte getSunlight(int x, int y, int z) {
     public byte getTotalLight(int x, int y, int z) {
         return 0;  //To change body of implemented methods use File | Settings | File Templates.
     }
-    
+
     @Override
     public int setExtraData(int index, Vector3i pos, int value) {
         Integer prevValue = getExtraDataLayer(index).put(pos, value);
         return prevValue == null ? 0 : prevValue;
     }
-    
+
+    @Override
+    public int setExtraData(int index, CoreChunk chunk, Vector3i pos, int value) {
```

Comment:
By convention, if a method takes a chunk, it should take a relative position, not absolute as here.